tests/e2e/uninstall.go

```diff
@@ -205,3 +208,15 @@ func requireRemoveCertManagerCRDs(kubectlOptions k8s.KubectlOptions) {
 		}
 	})
 }
+
+// requireRemoveNamespace deletes the indicated namespace object
+func requireRemoveNamespace(kubectlOptions k8s.KubectlOptions, namespace string) {
+	It(fmt.Sprintf("Removing namespace %s", namespace), func() {
+		err := deleteK8sResourceNoErrNotFound(kubectlOptions, defaultDeletionTimeout, "namespace", namespace, "--wait")
+		Expect(err).ShouldNot(HaveOccurred())
+
+		remainingResources, err := getK8sResources(kubectlOptions, []string{"namespaces"}, "", "", kubectlArgGoTemplateName)
```

Comment:
This check is unnecessary because the  deleteK8sResourceNoErrNotFound should fail if the namespace has not been deleted during the specified  defaultDeletionTimeout.

Reply:
It was there out of an abundance of caution but I see what you mean.

Fixed in 161a74729ecfb36787f3c5d541b58bd932b93f09.